modules/apps/blogs/blogs-service/src/main/java/com/liferay/blogs/service/impl/BlogsEntryLocalServiceImpl.java

```diff
@@ -270,65 +270,6 @@ public BlogsEntry addEntry(
 			smallImageImageSelector, serviceContext);
 	}
 
-	/**
-	 * @deprecated As of Cavanaugh (7.4.x), replaced by {@link
-	 * #addEntry(String, long, String, String, String, String, String, Date,
-	 * boolean, boolean, String[], String, ImageSelector, ImageSelector,
-	 * ServiceContext)}
-	 */
-	@Deprecated
-	@Indexable(type = IndexableType.REINDEX)
-	@Override
-	public BlogsEntry addEntry(
-			long userId, String title, String subtitle, String urlTitle,
-			String description, String content, Date displayDate,
-			boolean allowPingbacks, boolean allowTrackbacks,
-			String[] trackbacks, String coverImageCaption,
-			ImageSelector coverImageImageSelector,
-			ImageSelector smallImageImageSelector,
-			ServiceContext serviceContext)
-		throws PortalException {
-
-		return blogsEntryLocalService.addEntry(
-			null, userId, title, subtitle, urlTitle, description, content,
-			displayDate, allowPingbacks, allowTrackbacks, trackbacks,
-			coverImageCaption, coverImageImageSelector, smallImageImageSelector,
-			serviceContext);
-	}
-
-	/**
-	 * @deprecated As of Cavanaugh (7.4.x), replaced by {@link
-	 * #addEntry(String, long, String, String, String, String, String, int, int,
-	 * int, int, int, boolean, boolean, String[], String, ImageSelector,
-	 * ImageSelector, ServiceContext)}
-	 */
-	@Deprecated
-	@Override
-	public BlogsEntry addEntry(
```

Comment:
Did you check the usage of this method? I didn't find any code related to the replacement of this method calls in the PR changes. Maybe it was unused?